flight/mcu/lib/teensyShared/rk4.h

```diff
@@ -0,0 +1,181 @@
+#include <array>
+
+#include "Atmosphere.h"
+
+#pragma once
+
+using std::array;
+
+class rk4 {
+   public:
+    rk4();
+
+    array<float, 2> accel(array<float, 2> u, float rho);
+
+    array<float, 2> rk4_step(array<float, 2> state, float dt, float rho);
+
+    array<float, 2> sim_apogee(array<float, 2> state, float dt);
+
+    float cd(float alt, float vel);
+
+    array<double, 151> poly = {
+        {0.00000000000000000000000000000000000000000000000001,
+         -0.00000000000000000000000000000000000000000000000001,
+         -0.00000000000000000000000000000000000000000000000005,
+         -0.00000000000000000000000000000000000000000000000012,
+         -0.00000000000000000000000000000000000000000000000019,
+         -0.00000000000000000000000000000000000000000000000012,
+         0.00000000000000000000000000000000000000000000000054,
+         0.00000000000000000000000000000000000000000000000288,
+         0.00000000000000000000000000000000000000000000000927,
+         0.00000000000000000000000000000000000000000000002401,
+         0.00000000000000000000000000000000000000000000005278,
+         0.00000000000000000000000000000000000000000000009686,
+         0.00000000000000000000000000000000000000000000012938,
+         0.00000000000000000000000000000000000000000000002174,
+         -0.00000000000000000000000000000000000000000000065919,
```

Comment:
everything i've selected here truncates to 0 so it doesn't do anything in the calculation

Reply:
alright as long as the order of the polynomial doesnt change, this should be alright

Reply:
@parthshrotri so should I just replace whatever nick selected there with zeros? I don't see any point in leaving those there if they get truncated anyways

Reply:
that sounds fine to me